cmd/egress/cmd.go

```diff
@@ -217,5 +217,10 @@ are set correctly before execution.
 		os.Exit(1)
 	}
 
+	if err := validateEgressCmd.MarkFlagRequired("security-group-id"); err != nil {
```

Comment:
if this will be transformed into a required field, we can mention it on the cmd usage doc, too
```shell
▶./osd-network-verifier egress --subnet-id test
Error: required flag(s) "security-group-id" not set
Usage:
  osd-network-verifier egress [flags]

Examples:
For AWS, ensure your credential environment vars
AWS_ACCESS_KEY_ID, AWS_SECRET_ACCESS_KEY (also AWS_SESSION_TOKEN for STS credentials)
are set correctly before execution.

# Verify that essential OpenShift domains are reachable from a given SUBNET_ID
./osd-network-verifier egress --subnet-id $(SUBNET_ID)
```

and note that the error here is for reporting the registration error, not when the flag is not provided
so IMO the following would be more graceful (+ to prevent the usage of os.exit around)
```
	if err := validateEgressCmd.MarkFlagRequired("security-group-id"); err != nil {
		validateEgressCmd.PrintErr(err)
	}
```

Reply:
So, removed os.Exit(1)?